src/rsa_api/rsa_api.py

```diff
@@ -2406,12 +2406,95 @@ def IQSTREAM_StatusParser(
             if bool(status & 0x200000):  # mask bit 21
                 status_str += "Output buffer overflow. File writing too slow, "
                 status_str += "data loss has occurred.\n"
+            if bool(status & 0xFFC00000):
+                status_str += (
+                    "Invalid status code returned. Some always-zero bits are nonzero."
+                )
             if exit:
                 # Raise error with full string if configured
                 raise RSAError(status_str)
+        return status_str
+
+    @staticmethod
+    def IQSTREAMIQInfo_StatusParser(
+        iq_stream_info: _IQStreamIQInfo, exit: bool = True
+    ) -> Union[None, str]:
+        """
+        Parse an _IQStreamIQInfo struct to get the acquisition status.
+
+        Depending on the ``exit`` parameter, this method will either raise an
+        error or return a status string. Possible values for the
+        returned status string (when ``exit`` is False):
+
+        - No error
+        - Input overrange.
+        - USB data stream discontinuity.
+        - Input buffer > 75% full.
+        - Input buffer overflow. IQ Stream processing
+          too slow. Data loss has occurred.
+        - Output buffer > 75% full.
+        - Output buffer overflow. File writing
+          too slow. Data loss has occurred.
+        - Invalid status code returned. Some always-zero bits are nonzero.
+
+        In the case of multiple status codes being returned, the status
+        string will contain all returned status strings, separated by line
+        breaks.
+
+        Parameters
+        ----------
+        iq_stream_info : _IQStreamFileInfo
```

Comment:
This should be _IQStreamIQInfo

Reply:
Fixed in [4f31eb9](https://github.com/NTIA/tekrsa-api-wrap/pull/14/commits/4f31eb905604937550d5a4b3b726459a4568eded)